prow/ANNOUNCEMENTS.md

```diff
@@ -1,10 +1,10 @@
 DOCUMENTATION DEPRECATION NOTICE: This file is deprecated. Please refer to the
 [new migrated
-location](https://github.com/kubernetes-sigs/prow/tree/main/site/content/en/docs/Legacy%20Snapshot/prow/ANNOUNCEMENTS.md).
+location](https://docs.prow.k8s.io/docs/announcements/).
 Please do not edit this file; instead, make changes to the new location!
 
 The new location is served on the web at
-https://docs.prow.k8s.io/docs/legacy-snapshot/.
+https://docs.prow.k8s.io/docs/.
 
 This file will be deleted on 2022-11-30.
```

Comment:
If needed, we might extend the end-of-life of these tombstone files. 😊

Reply:
SGTM. This would give some more time for folks to adjust to the new updated layout in https://github.com/kubernetes-sigs/prow/pull/26 once that's merged. Maybe another 90 days? WDYT?

Reply:
I forgot to add: the deadline update should be a separate PR.

Reply:
> Maybe another 90 days? WDYT?

SGTM. 😊
I will open a separate PR to update the deadline.